### PR TITLE
Add example

### PR DESCRIPTION
**Pull request by pmantica1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,70 +57,68 @@ It's modeled after Python's `json.tool`, reading from stdin and writing to stdou
...
-   query language that we refer to as `MATCH`, after the name of its graph traversal operator.
-   With OrientDB, `MATCH` should be the preferred choice for most users, since it tends to run
-   faster than `gremlin`, and has other desirable properties. See the
-   [Execution model](#execution-model) section for more details.
-   
-   Support for relational databases including PostgreSQL, MySQL, SQLite,
-   and Microsoft SQL Server is a work in progress. A subset of compiler features are available for
-   these databases. See the [SQL](#sql) section for more details.
+## Features
+* **Databases and Query Languages:** We currently support a single database, OrientDB version 2.2.28+, and two query languages that OrientDB supports: the OrientDB dialect of gremlin, and OrientDB's own custom SQL-like query language that we refer to as MATCH, after the name of its graph traversal operator. With OrientDB, MATCH should be the preferred choice for most users, since it tends to run faster than gremlin, and has other desirable properties. See the Execution model section for more details.
```
Comment (bojanserafimov) [inline README.md]: The docs are a bit inconsistent. Here we say `We currently support a single database, OrientDB`, but later in the `SQL` section we say `Relational databases are supported by compiling to SQLAlchemy`. @jmeulemans should we update this section to include `SQL` as a backend, or do we wait for your branch to merge into master before we do that?

Comment (jmeulemans) [reply README.md]: There's an outstanding branch with the prototype SQL compiler, but that won't ever merge. The docs here are accurate, why are we removing the SQL details?

Comment (pmantica1) [reply README.md]: I am sorry I deleted the SQL details by mistake. Let me fix that.

**Comment by obi1kenobi** [comment]
Unless someone else has strong feelings otherwise, I'd probably keep the examples in the README for now, since I like having all info in one place when I open a new project's Github.

@pmantica1, I'm a bit confused about this PR though -- it seems to be making a lot of (seemingly to me) unrelated changes: changes in test setup code, in adding GraphQL syntax highlighting, in docstrings across the codebase, adding more content to the README, and reordering existing README content. For the sake of easier reviewing and more focused discussion, would you mind splitting up these changes into smaller PRs that do one thing at a time?

**Pull request closed without merging**